File: WebApi/api/src/main/java/pt/wastemanagement/api/model/CollectZone.java
package pt.wastemanagement.api.model;

public class CollectZone {
    public int collectZoneId, routeId, pickOrder;
    public String active; // T | F

    public CollectZone(int collectZoneId, int routeId, int pickOrder, String active) {
        this.collectZoneId = collectZoneId;
        this.routeId = routeId;
        this.pickOrder = pickOrder;
        this.active = active;
    }
}

File: Application/GabageApp/app/src/main/java/ps/isel/ipl/pt/gabageapp/model/CollectZone.kt
package ps.isel.ipl.pt.gabageapp.model

import com.google.android.gms.maps.model.LatLng

/**
 * Created by goncalo on 29/05/2018.
 */
class CollectZone(val collectZoneId: Int,
                  val routeId: Int,
                  val numContainers: Int,
                  val active: Boolean,
                  val location: LatLng,
                  val generalOccupation: Int,
                  val plasticOccupation: Int,
                  val paperOccupation: Int,
                  val glassOccupation: Int,
                  val containerUrl: String) {
}